### ISSUE TITLE
non-copular nominal should not have advcl dependent

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#375]
- [ ] [EWT](http://universal.grew.fr/?custom=635342e8a442b)
- [ ] [GUM](http://universal.grew.fr/?custom=635343193ec80)

**Comment by amir-zeldes** [comment]
I think all 4 GUM examples are correct.

**Comment by nschneid** [comment]
I now see 6 GUM examples. The first 2 are advcl modifiers of PP fragments, which is OK (no predicate to hang it on). The others:

- "the average [of the two middle values] [when the data are arranged in ascending order]": adnominal, so should be acl
- "the middle value [when the data are arranged in ascending order]": same
- "I'm starting to think like, what [if Gordon Ramsay is watching my vlogs and just absolutely shitting on me]": "what if" is documented as advcl on the [fixed page](https://universaldependencies.org/en/dep/fixed.html), so this is correct
- "If I'm sounding out of breath, [my lingering shortness of breath], and then also you know that I always talk fast in these videos.": this does not sound like a well-integrated modification to me, rather like a dysfluency or parataxis. Maybe dep?

**Comment by amir-zeldes** [comment]
I'm not sure these should be `acl` just because they attach to a noun. We allow `advmod` on a noun in:

  * President Biden's arrival just now/advmod

We don't want that to be `amod`, right? If so, then I think we should allow an analogous `advcl` for:

  * President Biden's arrival when it rained/advcl

I think retaining parity between clausal and nominal labels is important (and AFAIK it's also the major reason why a putative 'cobl' for oblique clausal arguments is labeled `advcl` rather than `ccomp`, right?). If we want to know whether a clause is attached to a noun, we can just check the parent POS tag. A 'when' clause should stay adverbial just like an advmod stays an advmod.

**Comment by nschneid** [comment]
There are indeed constructions where adverbs can modify a noun. The adverb category is heterogenous, so I'm not sure we can generalize this to clauses.

With a dependent clause, my understanding is that [acl](https://universaldependencies.org/u/dep/acl.html) vs. [advcl](https://universaldependencies.org/u/dep/advcl.html) is strictly based on whether it is modifying a nominal or not. From the `acl` page:

> The `acl` relation contrasts with the [advcl](https://universaldependencies.org/u/dep/advcl.html) relation, which is used for adverbial clauses that modify a predicate.

That it's a "when" clause suggests is relevant to knowing that it's an _adjunct_, but of course UD doesn't make the argument/adjunct distinction. Personally, I would be happy to change this whole approach to `cmod` vs. `cobl` or something like that, because the POS of the head is already listed separately (as you point out). But I don't see that happening before UDv3.

**Comment by nschneid** [comment]
The [advcl page](https://universaldependencies.org/u/dep/advcl.html#modifying-nominal-predicates) says directly that `advcl` only modifies predicates.

**Comment by amir-zeldes** [comment]
Yeah, that is pretty unambiguous... Can we have a chat about this in the group before I go and destroy this (albeit rare) distinction in our data? It just seems really odd to me that clauses get handled so differently from nominals here and I don't understand the motivation. Since it is not changing the deprel inventory, maybe there is a chance to change this or make it optional before V3. (I anticipate the answer will be 'that's the way it is', but I'd be remiss not to try)

**Comment by nschneid** [comment]
You're welcome to bring it up, but it looks like `acl` is used in such cases a majority of the time even in GUM: http://universal.grew.fr/?custom=63e2d5f49209b
